src/providers/data_providers/businessImageProviderAggregator.ts

```diff
@@ -2,16 +2,26 @@ import { BusinessImageProvider } from './businessImageProvider';
 import { ImageProviderName } from '../../models/data_models/image';
 import { MockBusinessImageProvider } from './mockBusinessImageProvider';
 import { YelpBusinessImageProvider } from './yelpBusinessImageProvider';
+import { OpenCageApiUtils } from '../../utils/openCageApiUtils';
 import { Image } from '../../models/data_models/image';
 import { BusinessInfoInput } from '../../models/data_models/types';
+import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
 
 export class BusinessImageProviderAggregator {
+	// load configs here
+	protected dataSourceConfigs: DataSourceConfiguration;
```

Comment:
is `dataSourceConfig` a better name?

Reply:
Hmm I was contemplating on the name last night because `DataSourceConfiguration` actually contains multiple configs from different sources.